frontend/src/app/shared/components/charts/charts.component.html

```diff
@@ -24,7 +24,7 @@
             </ngx-charts-tree-map>
         </div>
     </div>
-    <div *ngIf="multi?.length > 0 && options" class="single-container">
+    <div *ngIf="multi?.length > 0" class="single-container">
```

Comment:
I think Tanya want's just `*ngIf="multi?.length"`

Reply:
You didn't understand me. I updated my comments